app/assets/stylesheets/cclow/_events_timeline.scss

```diff
@@ -1,10 +1,13 @@
 @import 'variables';
 
+$arrow-btn-width: 48px;
+
 .events-container {
   display: flex;
   position: relative;
   overflow-y: hidden;
```

Comment:
I think it should be just `overflow: hidden` otherwise I have horizontal scrollbar, we should use navigation arrrows only.